frontend/src/component/UserApp.tsx

```diff
@@ -0,0 +1,14 @@
+import useUser from "../hook/useUser";
+import JobPostingGallery from "./JobPostingGallery";
+
+export default function UserApp() {
+
+    const {user} = useUser("1")
+
+
+    return (<div>
+        {user !== undefined && <JobPostingGallery user={user}/>}
+        {user===undefined&&<p>User is undefined.</p>}
+    </div>)
```

Comment:
Vorschlag (optional): Hier kannst Du die Klammern weglassen.
```suggestion
    return <div>
        {user !== undefined && <JobPostingGallery user={user}/>}
        {user===undefined&&<p>User is undefined.</p>}
    </div>
```

Reply:
👍🏻

Reply:
Wo wir schon in so Gebiete von einem Zeichenunterschied gehen fände ich persönlich ja am schönsten

```js 
 return (
       <div>
          {user !== undefined && <JobPostingGallery user={user}/>}
          {user===undefined&&<p>User is undefined.</p>}
      </div>
   )
      ```
      
 (Deshalb dann auch die Klammern) 
 Ist jetzt nur FYI ist fein wie es so jetzt steht